src/lib.rs

```diff
@@ -334,9 +349,18 @@ impl RestClient {
         T: serde::de::DeserializeOwned + RestPath<U>,
     {
         let req = self.make_request::<U, T>(Method::GET, params, Some(query), None)?;
-        let body = self.run_request(req).await?;
-
-        serde_json::from_str(body.as_str()).map_err(|err| Error::DeserializeParseError(err, body))
+        
+        #[cfg(feature = "lib-serde-json")]
+        {
+            let body = self.run_request(req).await?;
+            serde_json::from_str(body.as_str()).map_err(|err| Error::DeserializeParseError(err, body))
+        }
+        
+        #[cfg(feature = "lib-simd-json")]
+        {
+            let mut body = self.run_request(req).await?;
+            simd_json::serde::from_str(&mut body).map_err(|err| Error::DeserializeParseSimdJsonError(err, body))
```

Comment:
if you have a `body.as_bytes()` method you can speed this up as simd_json handles utf8 validation on the body and is more performant than rust's internal utf8 check (alone by the fact that using bytes the validation is done only once ;) )